eras/conway/impl/src/Cardano/Ledger/Conway/TxBody.hs

```diff
@@ -378,6 +382,8 @@ instance Crypto c => ConwayEraTxBody (ConwayEra c) where
     lensMemoRawType ctbrVotingProcedures (\txb x -> txb {ctbrVotingProcedures = x})
   proposalProceduresTxBodyL =
     lensMemoRawType ctbrProposalProcedures (\txb x -> txb {ctbrProposalProcedures = x})
+  currentTreasuryValueTxBodyL =
+    lensMemoRawType ctbrCurrentTreasuryValue (\txb x -> txb {ctbrCurrentTreasuryValue = x})
```

Comment:
Wondering why we don't `INLINE` these like the other lenses in `*EraTxBody`.

Reply:
Feel free to add them, it's good for performance. Whoever implemented these lenses simply have forgotten to add them.